src/fontra/backends/designspace.py

```diff
@@ -459,14 +474,22 @@ async def putGlyph(
             glyphSet = ufoLayer.glyphSet
             writeGlyphSetContents = glyphName not in glyphSet
 
+            customData = None
             if glyphSet == self.defaultUFOLayer.glyphSet:
                 layerGlyph = defaultLayerGlyph
                 storeInLib(layerGlyph, GLYPH_DESIGNSPACE_LIB_KEY, localDS)
                 storeInLib(layerGlyph, SOURCE_NAME_MAPPING_LIB_KEY, sourceNameMapping)
                 storeInLib(layerGlyph, LAYER_NAME_MAPPING_LIB_KEY, layerNameMapping)
-                storeInLib(layerGlyph, CUSTOM_DATA_LIB_KEY, glyph.customData)
+                customData = (
+                    glyph.customData | localSourcesCustomData[ufoLayer.fontraLayerName]
```

Comment:
We need to differentiate between these two, we can't just fold them into one. 

Suggest to use two keys:
```python
GLYPH_CUSTOM_DATA_LIB_KEY = "xyz.fontra.glyph.customData"
GLYPH_SOURCE_CUSTOM_DATA_LIB_KEY = "xyz.fontra.glyph.source.customData"
```

Reply:
Thank you very much for the valid feedback. I changed it.